### DISCUSSION TITLE
Consider a Managed Plan for Newbs

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [datalust/seq-tickets#1445] [Ideas]
Seq is amazing!  I am really surprised that the only onboarding options are ones you have to install and manage with your own devices.

Additionally, [based on this discussion](https://github.com/datalust/seq-tickets/discussions/1438#discussioncomment-1943912), the suggestion is to use Azure Premium Files.  But I am learning a little asterisk here, in that based on my observation of a week's worth of usage, that ends up being over $10/day!

(This could be due to my own newbness as I am also learning Azure as well -- in any case I am bumping that sucker down)

In any case, I would really like to see a managed solution from the Seq team, where they use their own install premises, fully secured/managed/configured by them, and all you have to do is log in and get your own subdomain to boot... something like $5-10/mo.  

I mean, we're already forking over cash to Azure anyways.  I would feel better giving that directly to the Seq team to help them out with their bottom line, and to support the cause in general.

Thank you for any consideration. 🙏

**Comment by Mike-E-angelo** [comment]
So FWIW the cost was actually around 6-dollars-per-day and the reason was that the size of the folder was at 1TB, which is wayyyyyy more than I needed.  After setting it to the minimum of 100GB (which is totally sus as the billing is based on drive size!), it is $.60/day or $18/mo.  Couple this with a App Service Hosting plan which is estimated to be around $10/mo, it's basically $30/mo for the most basic of diagnostics solution in Azure.  This seems a bit high to me, especially for a sole-developer outfit such as myself that is just getting off the ground.  I would gladly pay for $10/mo for a bare-bones hosted solution that doesn't involve so much configuration, is AAD-compatible, and "just works."

**Comment by liammclennan** [comment]
Hi Mike,

That's an interesting idea. Thanks for the suggestion.

**Comment by andymac4182** [comment]
@Mike-E-angelo Curious how much data would you like to ingest for $10 a month?

**Reply by Mike-E-angelo** [reply]
As much a freakin/reasonably possible @andymac4182. 😁  Reasonable to me is anything that helps an independent developer get off the ground.

**Reply by andymac4182** [reply]
The reason I ask is the simplest AWS set up I can build would be around 10USD for 100GB and 1 fairly small Fargate task. Not a lot of profit in there for the team. Happy to help you get a Fargate template set up that would deploy this.

**Reply by Mike-E-angelo** [reply]
I mean 100GB sounds like a _lot_, so that is a win.  However, consider that I am in the Microsoft for Startups plan currently and have all my Azure taken care of for the next year or so.  To add a little more clarity, my intention here was to get Seq/Datalust on the path to having a fully-managed solution.  And while I do appreciate the offer, I am not looking for alternatives at the moment.  In my case, I have already "eaten the pain" and after many trials/tribulations have my own instance setup via Azure Container Services.  The team there helped get situated, in fact, and I am grateful. 🙏  

Outside of that, I intended this request for Seq's/Datalust review and consideration for future offerings.  I am sure they can work with the margins/scale to get the desired profit to work. :)

**Reply by andymac4182** [reply]
From a curiosity sake as you are in Azure do you need your logs in Azure or would AWS be okay?

**Reply by Mike-E-angelo** [reply]
From my perspective, I have invested all my time in Azure so anything else at this point would not make a lot of sense (or cents 😁) to me.  However, I might have different feelings about it once my sponsorship runs out. :P

**Comment by Porkstone** [comment]
I would really like an option for a managed plan hosted on GCP next time you are thinking about the product roadmap

**Reply by liammclennan** [reply]
Hi @Porkstone, 

So I understand correctly, can you please clarify why it is important to you that the service be hosted in GCP?

**Reply by Porkstone** [reply]
GCP is where we host all our data so from a compliance point of view it would be simpler for us to be on one platform (less to understand).
If the managed service was on AWS or Azure that wouldn't be a deal breaker for us.
Main reason for a managed service is that we only have one part-time dev-ops resource and we won't be hiring more till sometime next year.

**Reply by liammclennan** [reply]
Thanks.

**Comment by Mike-E-angelo** [comment]
What I wouldn't give to have this right very now.  I can tell you it would be more than $10/mo. 😖  I had yet another instance today where Seq completely pooped the bed, and I have absolutely no idea the first place to look.  I am completely out of my element with 1) Azure 2) Linux Container and 3) Seq internals.  I am literally, once again, torpedoed without a motor.  

Please make this a priority. 🙏  I am at the mercy of Seq giving in at the slightest bump of traffic and it is quite unnerving. 😭

**Reply by Mike-E-angelo** [reply]
Update: created discussion here: https://github.com/datalust/seq-tickets/discussions/1651
